### ISSUE TITLE
Powers Info

### ISSUE THREAD

**Issue by Dhelindel** [torilmud/issues#13]
Powers info doesn't tell you which class is able to utilize the power. The old help file would advise you the class and circle. This would be helpful for for newbies and some oldies like me. 

Example the news said that a new alias was added for sonic wave. Powers info sonic doesn't say which classes are able to use that power. While help sonic wave tells you is a bard/bchanter 10th circle skill

**Comment by shevarash** [comment]
That's a valid complaint. I'll think about the best way to reflect this on the info screen without being too spammy.

**Comment by dnavarro** [comment]
For discussion:

You can only use powers info on powers that you can currently use, right?

Do we intend to phase out help files of spells? Otherwise don't help files and powers info have separate functions?

**Comment by shevarash** [comment]
No, anyone can use powers info on any power. If you have the power it will show you accurate stats instead of baseline stats. 

I would like to phase out help files for powers. I'm thinking that we could add a classes line to the powers info screen that looks a lot like the help files do now.
